veth-create-host.sh

```diff
@@ -0,0 +1,29 @@
+#! /bin/bash
+
+SERVER=$1
+CLIENT=$2
+
+if [ -z $SERVER ]; then
+    echo "Server NetNS must be specified."
+    exit 1
+fi
+
+PID=$(docker inspect --format '{{ .State.Pid }}' $SERVER)
+SERVER_NETNS="$SERVER_$PID"
+ln -s /proc/$PID/ns/net /var/run/netns/$SERVER_NETNS
+
+if [ ! -z $CLIENT ]; then
+    PID=$(docker inspect --format '{{ .State.Pid }}' $CLIENT)
+    CLIENT_NETNS="$CLIENT_$PID"
+    ln -s /proc/$PID/ns/net /var/run/netns/$CLIENT_NETNS
+fi
+
+for num in {1..32}; do
+    ip link add veth"$num" type veth peer name eth"$num" netns $SERVER_NETNS
+    ip netns exec $SERVER_NETNS ip link set eth"$num" up
+    ip link set veth"$num" up
+    if [ ! -z $CLIENT ]; then
```

Comment:
Not sure I understand what's the flow. Looks like the script can either link client with server or just create `veth` for server. In fact, any Docker provided as a single option - so, we should think about renaming `CLIENT`/`SERVER` into something like `NODE_A`/`NODE_B` or `NODE_1`/`NODE_2`. But what's the use-case? How we are going to use this node with external `veth` interfaces?

Reply:
I will add some "HOWTO" to the header.
The idea:
1. We create veth pairs to connect client and server.
2. The server must use isolated netns (default docker behavior).
3. The client can use isolated netns or run in host netns. The last option is useful when you run SAI-Challenger client with HW DUT or the external test environment that requires network connectivity.
4. I would keep SERVER and CLIENT as names, because the order is mandatory (I forgot why...).

Reply:
The items 1..3 - make sense. For the item 4, we should either add come clarification into the script or make it generic.

Also, we should not enforce the user to go though all the scripts one by one to understand in what cases and how they should be used. All scripts we have should be mentioned somewhere in the user documentation.

Reply:
Names are fixed.
This script is mentioned in client_server.md